sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/KeyVaultCertificateIdentifier.java

```diff
@@ -28,34 +28,32 @@
      *     <li>https://{key-vault-name}.vault.azure.net/deletedcertificates/{deleted-certificate-name}</li>
      * </ul>
      *
-     * @param id The identifier to extract information from.
+     * @param sourceId The identifier to extract information from.
      *
-     * @throws IllegalArgumentException If {@code certificateId} is an invalid Key Vault Certificate identifier.
-     * @throws NullPointerException If {@code certificateId} is {@code null}.
+     * @throws IllegalArgumentException If {@code sourceId} is an invalid Key Vault Certificate identifier.
+     * @throws NullPointerException If {@code sourceId} is {@code null}.
      */
-    public KeyVaultCertificateIdentifier(String id) {
-        if (id == null) {
-            throw new NullPointerException("'certificateId' cannot be null");
+    public KeyVaultCertificateIdentifier(String sourceId) {
+        if (sourceId == null) {
+            throw new NullPointerException("'sourceId' cannot be null");
         }
 
         try {
-            final URL url = new URL(id);
-            // We expect an identifier with either 2 or 3 path segments: collection + name [+ version]
+            final URL url = new URL(sourceId);
+            // We expect an sourceId with either 3 or 4 path segments: key vault + collection + name + "pending"/version
```

Comment:
nit for this comment in each Identifier (I think the brackets help to make it clear that the "pending" or version suffix is what's optional):
```suggestion
            // We expect a sourceId with either 3 or 4 path segments: key vault + collection + name [+ "pending"/version]
```